get-browser-globals.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const blacklist = [
+const denylist = [
```

Comment:
I don't think `deny` is the correct word here.
```suggestion
const ignorePatterns = [
```

Reply:
Makes sense to me!